### PR TITLE
upload cert cli

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: ctadata/cli.py [hunk 1]
```diff
@@ -52,6 +52,21 @@ def put_path(ctx, local_path, path, recursive):
...
+@click.argument("local_cert_path", type=click.Path(exists=True, dir_okay=False))
+@click.argument("local_ca_path", type=click.Path(exists=True, dir_okay=False))
+def upload_shared_certificate(ctx, local_cert_path, local_ca_path):
+    ctx.obj['api'].upload_shared_certificate(local_cert_path, local_ca_path)
+
+
+@cli.command("upload-certificate")
+@click.pass_context
+@click.argument("local_cert_path", type=click.Path(exists=True, dir_okay=False))
+def upload_shared_certificate(ctx, local_cert_path):
```
Comment (Ph0tonic) [inline ctadata/cli.py]:
```suggestion
def upload_certificate(ctx, local_cert_path):
```

**Review by Ph0tonic — CHANGES_REQUESTED** [review]

**Review by Ph0tonic — APPROVED** [review]
LGTM but we could maybe also rename `upload_certificate` in `api.py` to make more consistent.

**Comment by Ph0tonic** [comment]
> LGTM but we could maybe also rename `upload_certificate` in `api.py` to make more consistent.

Ok, you made all changes I wanted to suggest in between my review and my comment.

**Comment by volodymyrss** [comment]
> LGTM but we could maybe also rename `upload_certificate` in `api.py` to make more consistent.

yes, good point, I already pushed it.

**Comment by volodymyrss** [comment]
There is an issue still that 503 errors do do return json so API method itself crashes. But it's separate https://github.com/cta-epfl/ctadata/issues/21

Thanks!

**Pull request merged**